TSOClient/tso.client/UI/Panels/UIUCP.cs

```diff
@@ -409,15 +426,34 @@ public override void Update(FSO.Common.Rendering.Framework.Model.UpdateState sta
             }
 
             base.Update(state);
-
-            if (FSOEnvironment.Enable3D && Game.InLot)
+            if (Game.InLot)
             {
-                //if the zoom or rotation buttons are down, gradually change their values.
-                if (RotateClockwiseButton.IsDown) ((WorldStateRC)Game.vm.Context.World.State).RotationX += 2f / FSOEnvironment.RefreshRate;
-                if (RotateCounterClockwiseButton.IsDown) ((WorldStateRC)Game.vm.Context.World.State).RotationX -= 2f / FSOEnvironment.RefreshRate;
-                if (ZoomInButton.IsDown) Game.LotControl.TargetZoom = Math.Max(0.25f, Math.Min(Game.LotControl.TargetZoom + 1f / FSOEnvironment.RefreshRate, 2));
-                if (ZoomOutButton.IsDown) Game.LotControl.TargetZoom = Math.Max(0.25f, Math.Min(Game.LotControl.TargetZoom - 1f / FSOEnvironment.RefreshRate, 2));
+                if (state.NewKeys.Contains(Keys.F1) && !state.CtrlDown) SetPanel(1); // Live Mode Panel
```

Comment:
If we're going to be using this so often, it'd probably be better to get assign a local variable to `state.NewKeys`.